pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -125,6 +126,7 @@ type zedagentContext struct {
 	specMap                 types.ConfigItemSpecMap
 	globalStatus            types.GlobalStatus
 	getCertsTimer           *time.Timer
+	logMetrics              types.LogMetrics
```

Comment:
This doesn't look used.

Reply:
Yes, forgot to remove along with some other code. Will remove this.

Reply:
done